core/src/main/java/de/gerrygames/viarewind/protocol/protocol1_8to1_9/metadata/MetadataRewriter.java

```diff
@@ -47,12 +47,8 @@ public static void transform(Entity1_10Types.EntityType type, List<Metadata> lis
 							else entry.setValue(owner.toString());
 							break;
 						case BlockID:
-							int combined = (Integer) value;
-							int id = combined >> 4;
-							int data = combined & 0xF;
 							list.remove(entry);
-							list.add(new Metadata(metaIndex.getIndex(), MetaType1_8.Short, (short)id));
-							list.add(new Metadata(metaIndex.getIndex(), MetaType1_8.Byte, (byte)data));
+							list.add(new Metadata(metaIndex.getIndex(), MetaType1_8.Short, ((Integer) value).shortValue()));
```

Comment:
maybe this would be more prudent `((Number) value).shortValue()` ?

Reply:
A BlockID is always a VAR_INT